docs/Examples/Post/Main.purs

```diff
@@ -2,20 +2,11 @@ module Examples.Post.Main where
 
 import Prelude
 import Effect.Console (log)
-import HTTPure
-  ( Request
-  , ResponseM
-  , ServerM
-  , Method(Post)
-  , serve
-  , ok
-  , notFound
-  , toString
-  )
+import HTTPure (Method(Post), Request, ResponseM, ServerM, notFound, ok, serve, toString)
 
 -- | Route to the correct handler
 router :: Request -> ResponseM
-router { body, method: Post } = toString body >>= ok
+router request@{ method: Post } = toString request.body >>= ok
```

Comment:
I think you can revert this whole file too

Reply:
Reverted